fastlane/lib/fastlane/helper/git_helper.rb

```diff
@@ -12,6 +12,16 @@ def self.git_log_between(pretty_format, from, to, merge_commit_filtering)
       nil
     end
 
+    def self.git_log_last_commits(pretty_format, commit_count, merge_commit_filtering)
+      command = 'git log'
+      command << " --pretty=\"#{pretty_format}\" -n #{commit_count}"
+      command << " --no-merges" if merge_commit_filtering == :exclude_merges
```

Comment:
Can the `merge_commit_filtering` code be moved into a private method so that it isn't duplicated from lines 8 and 9?